source/connection-string/connection-string-spec.rst

```diff
@@ -180,9 +180,9 @@ The values in connection options MUST be URL decoded by the parser. The values c
 - Strings:
     The value
 - Integer:
-    The value parsed as a integer
+  The value parsed as a integer. If the value is the empty string, then a WARN level logging message MUST be issued and the key MUST be ignored. Drivers that cannot log warning messages or where warnings might be easily overlooked MAY raise an error.
```

Comment:
I'm having a hard time figuring how to apply the criteria of `where warnings might be easily overlooked`.  This doesn't seem specific to a driver, but rather to how logging is configured and what users do with their application log files, which is out of our control. Some applications upload logs to Splunk and alert on warnings, some log to /dev/null.  We just don't know.

I also think that leaving open this back door of allowing some drivers to error while most log goes against the goal of having portable connection strings.

Reply:
I would be fine just saying:

> If the value is the empty string, the key MUST be ignored

and leave the logging as an implementation detail.  I generally don't like having MUSTs about logging anyway, since we have no way of asserting the behavior.

Reply:
I have updated the language, removing the comments on logging.

Reply:
Well, I'm ok with it now, but I suspect other reviewers will not be! @jmikola, @p-mongo, would you like to weigh in?

Reply:
I'm OK with leaving it unspecified, but we could also consider "Drivers MAY log a warning to inform users that the key is being ignored." Will defer to you all on this, but spec LGTM as-is, too.

Reply:
I would prefer to leave it as-is.